Dockerfile.ubuntu

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:16.04
+
+#Base settings
+ENV HOME /root
+
+COPY requirements.txt /root/requirements.txt
+
+#Install ZeroNet
+RUN apt update \
+ && apt install -y python3-pip tor openssl \
+ gcc libffi-dev musl-dev make \
+ pkg-config \
+ && python3 -m pip install --user --upgrade pip
+
+RUN pip3 install -r /root/requirements.txt \
+ && echo "ControlPort 9051" >> /etc/tor/torrc \
+ && echo "CookieAuthentication 1" >> /etc/tor/torrc
+
+RUN python3 -V \
+ && python3 -m pip list \
+ && tor --version \
+ && openssl version
+
+#Add Zeronet source
+COPY . /root
+VOLUME /root/data
+
+#Control if Tor proxy is started
+ENV ENABLE_TOR false
+
+WORKDIR /root
+
+#Set upstart command
+CMD (! ${ENABLE_TOR} || tor&) && python3 zeronet.py --ui_ip 0.0.0.0 --fileserver_port 26552
+
+#Expose ports
+EXPOSE 43110 26552
```

Comment:
**You don't need to expose any port if you running it with Tor!** You could just allow in ufw:  `ufw allow in from yourclientip to serverip port 43110 proto tcp comment Zeronet`

Reply:
Why? I usually open ZN on host via 43110 port in browser :)

Reply:
> You don't need to expose any port if you running it with Tor!

Correction: that is only true if you're running ZeroNet in Tor: Always mode. That might be a good option, but again: it's a separate feature request; the other dockerfiles do this too.